packages/manager/modules/pci/src/projects/project/storages/databases/add/add.scss

```diff
@@ -1,3 +1,13 @@
 .order-summary {
   z-index: 1009;
 }
+
+oui-field.no-bottom-margin {
+  .oui-field {
+    margin-bottom: 0;
+  }
+}
+
+.oui-tile__description {
+  word-break: break-word;
+}
```

Comment:
**nitpick:** Please use scoped classes to avoid not anticipated changes at other places.

Reply:
Done here: https://github.com/ovh/manager/pull/8392/commits/ec34b726aa584b0c36591aee27bde3b6af0f7c95